css/index.css

```diff
@@ -101,7 +140,67 @@ p {
     justify-content: space-evenly;
 }
 
-.middle-content .boxes .box {
+.middle-content .boxes .box1 {
```

Comment:
One of crucial rules in CSS and overall in programming is `DRY` which means Do Not Repeat Yourself. You try to not write similar code more than once. In this example You are repeating all the rules for all the boxes everytime which breaks that rule and You should never do it.

What You should do is to set overall rules for `.boxes` or `.box` class and then just change individual colors for box1, box2, box3 etc WITHOUT repeating all these rules.

Example:

```
.middle-content .boxes .box {
    width: 12.5%;
    height: 10em;
    background: black;
    margin: 2% 2.5%;
    color: white;
    display: flex;
    align-items: center;
    justify-content: center;
}
.middle-content .boxes .box1 {
    background: red;
}
```

That way You specify rules for all the boxes once and then just change individual boxes background color by updating one rule, no reason to repeat all of them all the time because it is reduntant and bad for performance.